src/main/java/menion/android/whereyougo/gui/dialog/PositiveButtonActionCustomizableDialogFragment.java

```diff
@@ -34,7 +34,7 @@ public void onAttach(Context context) {
         super.onAttach(context);
 
         try {
-            listener = (NoPasswordDialogListener) context;
+            listener = (PositiveButtonActionCustomizableDialogListener) context;
         } catch (ClassCastException e) {
             throw new ClassCastException(getActivity().toString() + "must implement NoPasswordDialogListener.");
```

Comment:
Also rename here.

Reply:
What do you mean by that @bekuno?

Reply:
replace ` throw new ClassCastException(getActivity().toString() + "must implement NoPasswordDialogListener.");` 
by `throw new ClassCastException(getActivity().toString() + "must implement PositiveButtonActionCustomizableDialogListener.");`

Reply:
Fixed